playbooks/common/openshift-docker/config.yml

```diff
@@ -0,0 +1,85 @@
+- name: Set facts necessary to configure docker
+  hosts:
+    - masters
+    - nodes
```

Comment:
A lot of the config here seems like it should be encapsulated in a role as well.